### ISSUE TITLE
[3.3 rc7] Crash on VCS initialization

### ISSUE THREAD

**Issue by PoqXert** [godotengine/godot#47637]
**Godot version:**
3.3 rc7

**OS/device including version:**
Arch Linux

**Issue description:**
Download [git-plugin](https://github.com/godotengine/godot-git-plugin/wiki) and add it to new empty project. When try Setup Version Control and click "Initialize" button, Godot crashed with backtrace:
```
[1] /usr/lib/libc.so.6(+0x3cf80) [0x7f9eed556f80] (??:0)
[2] [0xb365f40] (??:0)
-- END OF BACKTRACE --
```
Godot 3.3 rc6 have't this bug.

**Steps to reproduce:**
- Create project
- Add [git-plugin](https://github.com/godotengine/godot-git-plugin/wiki)
- Go to `Project -> Version Control -> Set Up Version Control`
- Click "Initialize" button

**Minimal reproduction project:**
Any project

**Comment by akien-mga** [comment]
I can reproduce the issue with ad24eba47.

Debug stacktrace:
```
[1] /lib64/libc.so.6(+0x3aea0) [0x7f5fd24bbea0] (??:0)
[2] godot_variant godot::__wrapped_method<godot::GitAPI, bool, godot::String>(void*, void*, void*, int, godot_variant**) (??:0)
[3] NativeScriptInstance::call(StringName const&, Variant const**, int, Variant::CallError&) (/home/akien/Projects/godot/godot-3.x/modules/gdnative/nativescript/nativescript.cpp:709)
[4] Object::call(StringName const&, Variant const**, int, Variant::CallError&) (/home/akien/Projects/godot/godot-3.x/core/object.cpp:898 (discriminator 1))
[5] Object::call(StringName const&, Variant const&, Variant const&, Variant const&, Variant const&, Variant const&) (/home/akien/Projects/godot/godot-3.x/core/object.cpp:846)
[6] EditorVCSInterface::initialize(String) (/home/akien/Projects/godot/godot-3.x/editor/editor_vcs_interface.cpp:111 (discriminator 4))
[7] VersionControlEditorPlugin::_initialize_vcs() (/home/akien/Projects/godot/godot-3.x/editor/plugins/version_control_editor_plugin.cpp:135 (discriminator 2))
[8] MethodBind0::call(Object*, Variant const**, int, Variant::CallError&) (/home/akien/Projects/godot/godot-3.x/./core/method_bind.gen.inc:61 (discriminator 4))
[9] Object::call(StringName const&, Variant const**, int, Variant::CallError&) (/home/akien/Projects/godot/godot-3.x/core/object.cpp:919 (discriminator 1))
[10] Object::emit_signal(StringName const&, Variant const**, int) (/home/akien/Projects/godot/godot-3.x/core/object.cpp:1246 (discriminator 1))
[11] Object::emit_signal(StringName const&, Variant const&, Variant const&, Variant const&, Variant const&, Variant const&) (/home/akien/Projects/godot/godot-3.x/core/object.cpp:1304)
[12] BaseButton::_pressed() (/home/akien/Projects/godot/godot-3.x/scene/gui/base_button.cpp:135 (discriminator 2))
[13] BaseButton::on_action_event(Ref<InputEvent>) (/home/akien/Projects/godot/godot-3.x/scene/gui/base_button.cpp:174)
[14] BaseButton::_gui_input(Ref<InputEvent>) (/home/akien/Projects/godot/godot-3.x/scene/gui/base_button.cpp:64 (discriminator 3))
[15] MethodBind1<Ref<InputEvent> >::call(Object*, Variant const**, int, Variant::CallError&) (/home/akien/Projects/godot/godot-3.x/./core/method_bind.gen.inc:775 (discriminator 12))
[16] Object::call_multilevel(StringName const&, Variant const**, int) (/home/akien/Projects/godot/godot-3.x/core/object.cpp:761 (discriminator 1))
[17] Object::call_multilevel(StringName const&, Variant const&, Variant const&, Variant const&, Variant const&, Variant const&) (/home/akien/Projects/godot/godot-3.x/core/object.cpp:862)
[18] Viewport::_gui_call_input(Control*, Ref<InputEvent> const&) (/home/akien/Projects/godot/godot-3.x/scene/main/viewport.cpp:1716 (discriminator 2))
[19] Viewport::_gui_input_event(Ref<InputEvent>) (/home/akien/Projects/godot/godot-3.x/scene/main/viewport.cpp:2103 (discriminator 3))
[20] Viewport::input(Ref<InputEvent> const&) (/home/akien/Projects/godot/godot-3.x/scene/main/viewport.cpp:2923 (discriminator 2))
[21] Viewport::_vp_input(Ref<InputEvent> const&) (/home/akien/Projects/godot/godot-3.x/scene/main/viewport.cpp:1480)
[22] MethodBind1<Ref<InputEvent> const&>::call(Object*, Variant const**, int, Variant::CallError&) (/home/akien/Projects/godot/godot-3.x/./core/method_bind.gen.inc:775 (discriminator 12))
[23] Object::call(StringName const&, Variant const**, int, Variant::CallError&) (/home/akien/Projects/godot/godot-3.x/core/object.cpp:919 (discriminator 1))
[24] Object::call(StringName const&, Variant const&, Variant const&, Variant const&, Variant const&, Variant const&) (/home/akien/Projects/godot/godot-3.x/core/object.cpp:846)
[25] SceneTree::call_group_flags(unsigned int, StringName const&, StringName const&, Variant const&, Variant const&, Variant const&, Variant const&, Variant const&) (/home/akien/Projects/godot/godot-3.x/scene/main/scene_tree.cpp:276)
[26] SceneTree::input_event(Ref<InputEvent> const&) (/home/akien/Projects/godot/godot-3.x/scene/main/scene_tree.cpp:432 (discriminator 6))
[27] InputDefault::_parse_input_event_impl(Ref<InputEvent> const&, bool) (/home/akien/Projects/godot/godot-3.x/main/input_default.cpp:429)
[28] InputDefault::parse_input_event(Ref<InputEvent> const&) (/home/akien/Projects/godot/godot-3.x/main/input_default.cpp:273)
[29] InputDefault::flush_accumulated_events() (/home/akien/Projects/godot/godot-3.x/main/input_default.cpp:692)
[30] OS_X11::process_xevents() (/home/akien/Projects/godot/godot-3.x/platform/x11/os_x11.cpp:2337)
[31] OS_X11::run() (/home/akien/Projects/godot/godot-3.x/platform/x11/os_x11.cpp:3639)
[32] godot-3.x(main+0xfd) [0x17066df] (/home/akien/Projects/godot/godot-3.x/platform/x11/godot_x11.cpp:57)
[33] /lib64/libc.so.6(__libc_start_main+0xcd) [0x7f5fd24a87dd] (??:0)
[34] godot-3.x(_start+0x2a) [0x170653a] (??:?)
-- END OF BACKTRACE --
Aborted (core dumped)
```

**Comment by akien-mga** [comment]
It seems to have been fixed in f64d9423a577abc221474812764b53c95f56db06 (#47623).
